src/Legacy/Drupal7.php

```diff
@@ -43,7 +43,14 @@
 class Drupal7 {
 
   public function __call($method, $args) {
-    return call_user_func_array($method, $args);
+    // Hack: See http://goo.gl/aM5fra
```

Comment:
Lets have this say something like:

// This is a necessary hack to ensure that things that have been passed by reference continue to being passed by reference. For more information see: ...